src/skills-builder/data/reducer.js

```diff
@@ -0,0 +1,28 @@
+import {
+  SET_GOAL,
+  SET_JOB_TITLE,
+} from './constants';
+
+export function skillsReducer(state, action) {
```

Comment:
I know Reducer is a react term but in the context of "Skills Builder" this is pretty funny